pkg/curatedpackages/packageclient_test.go

```diff
@@ -0,0 +1,59 @@
+package curatedpackages_test
+
+import (
+	"context"
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	packagesv1 "github.com/aws/eks-anywhere-packages/api/v1alpha1"
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+)
+
+type packageTest struct {
+	*WithT
+	ctx        context.Context
+	bundle     *packagesv1.PackageBundle
+	command    *curatedpackages.PackageClient
+	customName string
+}
+
+func newPackageTest(t *testing.T) *packageTest {
+	return &packageTest{
+		WithT: NewWithT(t),
+		ctx:   context.Background(),
+		bundle: &packagesv1.PackageBundle{
+			Spec: packagesv1.PackageBundleSpec{
+				Packages: []packagesv1.BundlePackage{
+					{
+						Name: "harbor-test",
+					},
+					{
+						Name: "redis-test",
+					},
+				},
+			},
+		},
+		customName: "my-",
```

Comment:
I would reuse the variable defined in the other package, even if you need to export it.